pkg/config/testdata/application/ecs-app-service-discovery.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: pipecd.dev/v1beta1
+kind: ECSApp
+spec:
+  input:
+    serviceDefinitionFile: /path/to/servicedef.yaml
+    taskDefinitionFile: /path/to/taskdef.yaml
+    accessType: SERVICE_DISCOVERY
```

Comment:
This constant is defined by us or by AWS lib? 🤔

Reply:
"This constant" means 'SERVICE_DISCOVERY'?

If so, it's defined by us, not by AWS.

And other names would be added in the future, like 'APP_MESH'

Reply:
Then we should define these values as constant of ECS platform provider and have validate in config package

Reply:
thanks, I got it.  I'll fix it.

Reply:
@khanhtc1202 
I did it. Could you check again?
919cc3cf26b37b119563dc30614ca224f8a1d16e